ceph-releases/jewel/opensuse/Leap_42.2/base/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# CEPH BASE IMAGE on OpenSUSE Leap 42.2
+# CEPH VERSION: Jewel
+# CEPH VERSION DETAIL: 10.2.4
+
+FROM opensuse:42.2
+MAINTAINER Ricardo Dias "rdias@suse.com"
+
+# Install Ceph
+RUN zypper --no-gpg-checks -n ref
+RUN zypper -n --no-gpg-checks install net-tools
+RUN zypper -n --no-gpg-checks install ceph=10.2.4+git.1481215985.12b091b-4.1 ceph-radosgw=10.2.4+git.1481215985.12b091b-4.1
+RUN zypper -n --no-gpg-checks install rbd-mirror=10.2.4+git.1481215985.12b091b-4.1
```

Comment:
Why are you specifying --no-gpg-checks here - I'm a docker novice, but shouldn't "FROM opensuse:42.2" pull in the openSUSE repository key to allow for verification here?